packages/core/src/types/oas3_1.ts

```diff
@@ -181,6 +181,18 @@ const Schema: NodeType = {
   extensionsPrefix: 'x-',
 };
 
+const SchemaProperties: NodeType = {
+  properties: {
+    propertyName: (value: any) => {
+      if (typeof value === 'boolean') {
+        return { type: 'boolean' };
+      } else {
+        return 'Schema';
+      }
+    },
+  },
```

Comment:
I'm not 💯 sure, but it looks like it should be this way:

```suggestion
  properties: {},
  additionalProperties: (value) => {
      if (typeof value === 'boolean') {
        return { type: 'boolean' };
      } else {
        return 'Schema';
      }
  },
```

Could you check if that works better?

Reply:
how can I run the smoke tests locally?  if i run jest, the tests pass.

Do I need to commit the updated snapshots for the `build and test` command to pass?

```shell
Snapshot Summary
 › 270 snapshots updated from 60 test suites.

Test Suites: 1 skipped, 93 passed, 93 of 94 total
Tests:       2 skipped, 640 passed, 642 total
Snapshots:   270 updated, 84 passed, 354 total
Time:        47.02 s
```

Reply:
@tatomyr  i fixed it per your suggestion.  ok to merge now if you agree with the changes. thanks for the help!